ballerina-test/src/test/resources/grpc/expected-files/route_guide_pb.bal

```diff
@@ -0,0 +1,364 @@
+import ballerina/grpc;
```

Comment:
license header

Reply:
We do not add a license header here since this file is used to compare the output of the generated file